runtime/common/src/crowdloan.rs

```diff
@@ -855,6 +863,12 @@ mod tests {
 	fn bids() -> Vec<BidPlaced> {
 		BIDS_PLACED.with(|p| p.borrow().clone())
 	}
+	fn vrf_delay() -> u64 {
```

Comment:
unfortunately no one ever uses them 😁 but I added some pretty convenient ways to create all of these thread local stuff with `parameter_types!` with getter and setters https://crates.parity.io/frame_support/macro.parameter_types.html

Reply:
Actually i do normally use it, but copy paste from local code is faster than looking up the syntax lol